app/components/Story.js

```diff
@@ -212,17 +212,12 @@ class Story extends Component {
   }
 
   getIconStyles() {
-    if (this.state.isSplit) {
-      return [
-        {top: MID_ICON_HEIGHT},
-        styles.icon,
-      ];
-    } else {
-      return [
-        {top: 0},
-        styles.icon,
-      ];
-    }
+    const top = this.state.isSplit ? MID_ICON_HEIGHT : -4;
```

Comment:
changed from 0...

![fullsizerender](https://cloud.githubusercontent.com/assets/224622/25604333/844d3ee4-2eb7-11e7-8d20-4221bb37cb31.jpg)

to -4...

![fullsizerender 3](https://cloud.githubusercontent.com/assets/224622/25604339/8812bc5c-2eb7-11e7-9904-5c5a42cf8dfb.jpg)